_unittest/test_00_downloads.py

```diff
@@ -58,3 +58,7 @@ def test_download_3dcomp(self):
     def test_download_twin_builder_data(self):
         example_folder = self.examples.download_twin_builder_data("Ex1_Mechanical_DynamicRom.zip", True)
         assert os.path.exists(example_folder)
+
+    def test_download_folder(self):
+        example_folder = self.examples.download_file("motorcad", "IPM_Vweb_Hairpin.mot")
```

Comment:
is this method downloading a file only or it can download a directory with all the content?

Reply:
This is using the private method "_download_file", so this method is just downloading a file. Do you think is useful to implement this additional feature?

Reply:
yes, mainly for aedb. I think you can reuse some already available in some of the specific download methods

Reply:
Maybe we can create a download_folder method? (a separate one)